examples/managed-node-groups/main.tf

```diff
@@ -0,0 +1,468 @@
+#------------------------------------------------------------------------
```

Comment:
Also consider breaking this file into multiple files.

Reply:
I think the whole idea is to show all possible options to deploy managed node groups in one example hence keeping it in one file.